start.sh

```
@@ -2376,13 +2376,22 @@ start_services() {
     # 6. Start WeWork (run in background)
     if [ "$start_wework" = true ]; then
         echo -e "  Starting ${BLUE}wework${NC}..."
+
+        # Kill any existing wework process on the port first
+        local old_wework_pids=$(get_port_listener_pids "$WEWORK_PORT" 2>/dev/null | tr '\n' ' ')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)